app/controllers/Application.java

```diff
@@ -503,8 +532,15 @@ public static Result flowHistory() {
     }
 
     if (!Utils.isSet(partialFlowDefId)) {
-      return ok(flowHistoryPage.render(partialFlowDefId, graphType, flowHistoryResults.render(null, null, null, null)));
+      if (version.equals(VERSION_NEW)) {
+        return ok(newFlowHistoryPage
```

Comment:
it's better to name the new one as flowHistoryPage and the old one as oldFlowHistoryPage so that you dont have to rename again when you remove old.